kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/LessonContentCard/index.vue

```diff
@@ -10,30 +10,35 @@
     />
 
     <div :class="windowIsSmall ? 'mobile-text' : 'text'" :style="{ color: $themeTokens.text }">
-      <h3
-        v-if="!windowIsSmall"
-        class="title"
-        :class="{ 'has-message': Boolean(message) }"
-        dir="auto"
-      >
-        <KLabeledIcon :label="title">
-          <template #icon>
-            <ContentIcon :kind="kind" />
-          </template>
-        </KLabeledIcon>
-      </h3>
-      <h3
-        v-if="windowIsSmall"
-        dir="auto"
+      <div 
+        :class="{ 'title-message-wrapper': Boolean(!windowIsSmall) }" 
+        :style="{ color: $themeTokens.text }"
       >
-        <KLabeledIcon :label="title">
-          <template #icon>
-            <ContentIcon :kind="kind" />
-          </template>
-        </KLabeledIcon>
-      </h3>
-      <div v-if="message" class="message" :style="{ color: $themeTokens.text }">
-        {{ message }}
+        <h3
+          v-if="!windowIsSmall"
+          class="title"
+          :class="{ 'has-message': Boolean(message) }"
```

Comment:
It doesn't look like the `has-message` class exists anymore. A cleanup maybe?

Reply:
Good catch! Thanks for this.